npm/grep/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "@cypress/grep",
+  "version": "0.0.0-development",
+  "description": "Filter tests using substring",
+  "main": "src/support",
+  "scripts": {
+    "cy:run": "node ../../scripts/cypress.js run --config specPattern='**/unit.js'",
+    "cy:open": "node ../../scripts/cypress.js open --e2e -b electron --config specPattern='**/unit.js'"
+  },
+  "dependencies": {
+    "debug": "4.3.4",
```

can we open this up to this aligns with the other debug versions in the repo? or is it locked down for a reason?
```suggestion
    "debug": "^4.3.2",
```